src/solvers/sat/satcheck.h

```diff
@@ -48,6 +77,13 @@ typedef satcheck_minisat1t satcheck_no_simplifiert;
 typedef satcheck_minisat_simplifiert satcheckt;
 typedef satcheck_minisat_no_simplifiert satcheck_no_simplifiert;
 
+#elif defined SATCHECK_IPASIR
+
+#include "satcheck_ipasir.h"
+
+typedef satcheck_ipasirt satcheckt;
```

Comment:
This will only work with `--no-sat-preprocessor` (or `--stop-on-fail` in most cases).